.github/workflows/markdown-quality.yml

```
@@ -0,0 +1,158 @@
+      uses: actions/checkout@v4
+      
+    - name: Setup Node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version: '18'
+        
+    - name: Install markdownlint
+      run: npm install -g markdownlint-cli2
+      
+    - name: Collect quality metrics
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2129:style:19:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)